NEWS.md

```diff
@@ -16,6 +16,11 @@ This release is in development. For a stable release install 1.3.5 from CRAN.
   instead of `create_gp_data()`. By @jamesmbaazam in #388 and reviewed by @seabbs.
 * Improved compilation times by reducing the number of distinct stan models and deprecated `tune_inv_gamma()`. By @sbfnk in #394 and reviewed by @seabbs.
 * Changed touchstone settings so that benchmarks are only performed if the stan model is changed. By @sbfnk in #400 and reviewed by @seabbs.
+* [pak](https://pak.r-lib.org/) is now suggested for installing the developmental
```

Comment:
same issue as the other PR. Linebreaks are not needed.

Reply:
Noted. I've fixed it here f94184d